<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="button_tampil_form2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Form Pengguna</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>FORM MENU UTAMA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_tampil_form3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Form Pesanan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_tampil_form4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Form Produk</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
